.github/workflows/test.yml

```diff
@@ -53,9 +62,6 @@ jobs:
           path: fixture/zomes/foo/target
           key: ${{ matrix.os }}-build-${{ env.cache-name }}-${{ hashFiles('fixture/zomes/foo/Cargo.lock') }}
 
-      - name: Clean up nix files to prevent insufficient disk space
-        run: nix-collect-garbage -d
```

Comment:
I take it this is no longer required with the extra space?

Reply:
Exactly that